kv.go

```diff
@@ -401,6 +402,7 @@ func (js *js) CreateKeyValue(cfg *KeyValueConfig) (KeyValue, error) {
 		MaxMsgs:           -1,
 		MaxConsumers:      -1,
 		AllowDirect:       true,
+		RePublish:         cfg.RePublish,
```

Comment:
Humm... I did not test the situation where we call that on an existing stream. We may then have the issue I was discussing during the call at line 425 (where we compare the config that we got from a lookup vs what we are trying to set...). Will see if I can add a test to simulate the upgrade.

Reply:
Well, maybe that's ok then that the CreateKeyValue fails, the user should then administratively update the stream (if at all possible, not sure if we support updating this). Still will add a test.